src/controllers/dyalna.js

```diff
@@ -5,14 +5,29 @@ const news = (bot, msg) => {
   superagent
     .get('https://dyalna.com/api/post/top')
     .then(data => {
-      bot.startConversation(msg, (err,convo) => {
+      bot.startConversation(msg, (err, convo) => {
         convo.say('Sraw bezzaf swala7, voici les top news :')
         data.body.forEach(post => {
-          convo.say({ text: ` - ${ post.title } https://dyalna.com/post/show/${ post.slug }`, unfurl_links: false })
+          convo.say({
+            'attachments' : [{
+              'fallback': 'Required placeholder for news summary.',
```

Comment:
as it seems over the docs, its in a way or another a required field but apparently no harm on deleting it ! so yeah let's drop it !